scipy/ndimage/filters.py

```diff
@@ -806,13 +851,32 @@ def uniform_filter(input, size=3, output=None, mode="reflect",
     %(cval)s
     %(origin)s
 
+    Returns
+    -------
+    uniform_filter : ndarray
+        Return of same shape as `input`.
```

Comment:
This sentence could be clearer. I see that e.g. `gaussian_filter` says this is "Returned array of same shape as `input`", which could probably be further improved. Maybe something like:

 "Filtered array. Has the same shape as `input`."